### ISSUE TITLE
Cannot find the database creation file in the repository

### ISSUE THREAD

**Issue by prajwal1210** [karthikaS03/PushAdMiner#2]
Hi!

I am trying to run this code based on the README but there is no database creation as mention in the steps to run. Can you point me to the correct file to use to create a database?

**Comment by karthikaS03** [comment]
Added to the database directory. I had to recreate this file from a backup file. Let me know if there's any issues.

**Comment by prajwal1210** [comment]
Thank you so much for the change. Also, in api_requests:

```
dir_path = '../seed_lists/'
insert_api_url = 'https://oyi4twvq77.execute-api.us-east-2.amazonaws.com/default/notification-db-add?'
fetch_api_url = 'https://2zbo3zz9q9.execute-api.us-east-2.amazonaws.com/default/notification-db-insert?'
update_api_url = 'https://4h39mccr97.execute-api.us-east-2.amazonaws.com/default/notification-db-update?'

fetch_score_api = 'https://g6ekhqhjo1.execute-api.us-east-2.amazonaws.com/default/notification_db_score_fetch?'
```

There is reference to these URLs. How do you set this DB up?

**Comment by karthikaS03** [comment]
Don't worry about the API setup. you can setup DB on your own server.  I don't think this is referred to in the main code. you can comment the code related to this or make it point to your DB.

**Comment by prajwal1210** [comment]
The fetch_urls and update_url methods are which I am guessing make changes to the seed list db. Just want to know if that's all the functionality that api_requests.py provide.

**Comment by karthikaS03** [comment]
Yeah. we hosted it on AWS to keep track of the updates to Seed URL table across many servers running the containers.

**Comment by prajwal1210** [comment]
`DataCollector/api_calls/api_requests.py` - This particular file I mean

**Comment by prajwal1210** [comment]
Oh okay got it. Do you have a procedure to create this database and API as well? It seems different from the malvert_db in the database folder
